docs/03-github/02-activation.mdx

```diff
@@ -37,20 +37,24 @@ Create a file called `.github/workflows/activation.yml` and add the below workfl
 
 ```yaml
 name: Acquire activation file
+
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
```

Comment:
the object braces were there to make it less ambiguous for people that the key has no explicit content.